make.py

```diff
@@ -170,6 +170,17 @@ def tidy():
 def black():
     """Check code with the 'black' formatter."""
     print("\nblack")
+    # Black is no available in Python 3.5, in that case let the tests continue
+    try:
+        _ = subprocess.run([TIDY, "--version"])
```

Comment:
No need to assign the result of `subprocess.run` if it's not being used. (Just do `subprocess.run(..)`

Reply:
Yeah, it's a bit cleaner that way, originally I was looking at the output, but ultimately we only care if it throws an exception.

Reply:
assigning to _ is also a bit ugly, when we use _ for localization :)